testsuite/python/accumulator.py

```diff
@@ -29,6 +29,9 @@
 import espressomd.accumulators
 
 
+N = 4
```

Comment:
I would prefer `N_PART` for consistency with other tests, and because it's simpler to grep a word than a single character.